app/Observers/WarehouseStockTransactionsObserver.php

```
@@ -25,18 +26,23 @@ public function created(WarehouseStockTransactions $warehouseStockTransactions):
             $warehouseSummary->warehouse_id = $warehouseStockTransactions->warehouse_id;
             $warehouseSummary->item_id = $warehouseStockTransactions->item_id;
             $warehouseSummary->uom_id = $warehouseStockTransactions->uom_id;
+            $warehouseSummary->quantity = 0;
             $warehouseSummary->metadata = [
                 'last_transaction_id' => $warehouseStockTransactions->id
             ];
         }
         // get the new quantity based on the transaction type
         // and convert it to the summary's UOM if necessary
-        $summaryUom = $warehouseSummary->uom_id;
+        $summaryUom = UOM::find($warehouseSummary->uom_id);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\UOM' in method 'created'. (Clean Code Rules)

(StaticAccess)